### ISSUE TITLE
Inability to see important errors when developing in clojure.

### ISSUE THREAD

**Issue by golanw** [cursive-ide/cursive#1912]
I can see in the editor itself (when editing a specific file) that a function is underlined with a red line for a serious error - for example: Incorrect Arity.
However it's easy to miss things like that.
As a clojure dev team we need a proper list somewhere.
I tried to use code inspection, and indeed code inspection contains clojure inspections (including "Incorrect Arity" - see attached).
However when performing code inspection using just the clojure inspections, I get no errors at all! (see attached).
The errors do appear as part of the _General-->Annotator_ inspection (see attached), together with a lot of other non-important warnings (see attached - note that in our real dev project there are tons of meaningless stuff that "hides" this real error).

Please fix this asap.
Thanks.

**Clojure Inspection:**
![screen shot 2017-12-05 at 18 55 08](https://user-images.githubusercontent.com/33827830/33619711-9987edb2-d9ee-11e7-91c5-dc590d4ea23a.png)
![screen shot 2017-12-05 at 19 02 32](https://user-images.githubusercontent.com/33827830/33619848-e40c199e-d9ee-11e7-9767-de52c4498e0f.png)

**All Inspections:**
![screen shot 2017-12-05 at 18 55 45](https://user-images.githubusercontent.com/33827830/33619712-99abad9c-d9ee-11e7-8bcd-4cad9c23771a.png)
![screen shot 2017-12-05 at 19 01 52](https://user-images.githubusercontent.com/33827830/33619847-e3edc07a-d9ee-11e7-97a8-7d28be369003.png)

**Comment by otwieracz** [comment]
Yes, that's pity that surprisingly many errors are found by „General > Annotator”. However, things like spelling could be disabled separately.

Also, do you know how to use „resolutions”? This allows Cursive to have better clue how to treat specific function. For example, this `defresource` by default produces a lot of errors about for example undefined variable in `[foo]`

But, by telling Cursive to resolve `defresource` as `defn` it know that this `[foo]` is variable declaration part:

![Screenshot from 2019-06-13 07-08-43](https://user-images.githubusercontent.com/108072/59405257-27035f80-8daa-11e9-99cd-10c8b142cf34.png)

![Screenshot from 2019-06-13 07-08-57](https://user-images.githubusercontent.com/108072/59405258-27035f80-8daa-11e9-8959-f40ca9859cb3.png)

Unfortunately seems like this does not allow to workaround issues like with `quartzite`:
![Screenshot from 2019-06-13 07-51-04](https://user-images.githubusercontent.com/108072/59406940-08a06280-8db0-11e9-8b5d-151bfff9bec9.png)

Where without macroexpansion you have no way of knowing that there is additional variable carried and I don't see anything matching with similar syntax.

Maybe that's the question to authors - how to define my own resolutions?
